net/strongswan/files/swanctl.init

```diff
@@ -227,26 +227,34 @@ config_child() {
 	local updown
 	local firewall
 	local remote_subnet
-	local lifetime
+	local life_time
```

Comment:
Likewise, if it's preferred to combine all the local down below with the additions, let me know.